Godeps/Godeps.json

```diff
@@ -104,6 +104,10 @@
 			"Rev": "ee2372b58cee877abe07cde670d04d3b3bac5ee6"
 		},
 		{
+			"ImportPath": "github.com/rakyll/pb",
```

Comment:
@akolb1 @yichengq Do you know where do we have this dependency? I am not aware of it.